setup.py

```diff
@@ -82,6 +82,6 @@ def find_version():
         'dev': ['flake8'],
         'test': ['tox', 'pytest', 'coverage'],
         # recommonmark: to make sphinx render markdown
-        'doc': ['sphinx', 'recommonmark'],
+        'doc': ['sphinx', 'recommonmark', 'pylint'],
```

Comment:
Based on using pyreverse in the Makefile, I'm guessing this is supposed to be pyreverse, not pylint.

Could easily be wrong.